test/README.md

```diff
@@ -9,8 +9,8 @@ The following command assumes that CKB binary is built as `../target/release/ckb
 cargo run
 ```
 
-Run single spec:
+Run specified specs:
 
 ```bash
-cargo run ../target/debug/ckb 9000 spec_name
+cargo run -- --bin ../target/debug/ckb spec1 spec2
```

Comment:
how to specify the port?

Reply:
https://github.com/nervosnetwork/ckb/pull/1223/files#diff-f46cfe51c4764acc308811bb8b90300aR127
`--port`